tests/communities/test_services.py

```diff
@@ -67,6 +67,7 @@ def test_search_featured(community_service, comm, db, es_clear):
     hits = featured_comms["hits"]["hits"]
     assert len(hits) == featured_comms["hits"]["total"] == 1
     assert hits[0]["id"] == comm.data["id"]
+    assert "links" in hits[0]
```

Comment:
could you add a check for the featured link, if it is added to the response?

Reply:
The featured link (e.g. /api/community/<id>/featured) is not exposed, because the community links are used, which do not contain this link. IIRC, the idea was to just return the communities as usual, without any additional information.

https://github.com/inveniosoftware/invenio-communities/issues/497